app/addons/auth/components.react.jsx

```diff
@@ -239,9 +238,6 @@ define([
             <input id="password" type="password" name="password" placeholder= "Password" size="24"
               onChange={this.onChangePassword} />
             <p>
-              Non-admin users have read and write access to all databases, which
-              are controlled by validation functions. CouchDB can be configured to block all
-              access to anonymous users.
```

Comment:
I wanted to change the last sentence to "CouchDB can be configured to block the access of anonymous users." and add the link "http://docs.couchdb.org/en/1.6.1/config/auth.html" to it, but I found that it’s not necessarily related to ADMIN privileges, so I deleted the last sentence.

https://wiki.apache.org/couchdb/Security_Features_Overview